### ISSUE TITLE
Crash when downloading specific shows

### ISSUE THREAD

**Issue by cemrich** [cemrich/zapp#83]
This does not happen often and has to be reproduced before fixing:

```
java.lang.IllegalArgumentException: Can only download HTTP/HTTPS URIs:
at android.app.DownloadManager$Request.<init>(DownloadManager.java:617)
at de.christinecoenen.code.zapp.app.mediathek.ui.detail.MediathekDetailFragment.a(MediathekDetailFragment.java:165)
at de.christinecoenen.code.zapp.app.mediathek.ui.detail.MediathekDetailFragment.onDownloadMediumClick(MediathekDetailFragment.java:122)
at de.christinecoenen.code.zapp.app.mediathek.ui.detail.MediathekDetailFragment_ViewBinding$4.a(MediathekDetailFragment_ViewBinding.java:80)
at butterknife.a.a.onClick(DebouncingOnClickListener.java:22)
at android.view.View.performClick(View.java:6261)
at android.view.View$PerformClick.run(View.java:23752)
at android.os.Handler.handleCallback(Handler.java:751)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:154)
at android.app.ActivityThread.main(ActivityThread.java:6776)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1518)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

**Comment by cemrich** [comment]
I cannot reproduce this right now.

**Comment by cemrich** [comment]
This issue should be fixed in Zapp 1.11.0.
